packages/components/src/alignment-matrix-control/types.ts

```diff
@@ -54,6 +54,8 @@ export type AlignmentMatrixControlIconProps = Pick<
 	 * component instead_
 	 *
 	 * The size of the icon.
+	 *
+	 * @deprecated
```

Comment:
What happens in the generated README if a prop is `@deprecated` but not `@ignored` ? Should we somehow add that information ?

Reply:
I can't recall if we have any of these anymore. We can see when we encounter one. Maybe the answer will be to `@ignore` it.